### PR TITLE
Prepare release 0.2.0

### PR DESCRIPTION
**Pull request by abtris**
- drop node 0.8 support
- add gulp for testing
- add nodejs 0.12 support

### CODE REVIEW

**Comment by abtris** [comment]
This is last changes before bump to 0.2.0

**Comment by abtris** [comment]
@Almad Can you look at this changes and after merge I release new version.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
 ## s3-streaming-upload [![Build Status](https://travis-ci.org/apiaryio/s3-streaming-upload.png?branch=master)](https://travis-ci.org/apiaryio/s3-streaming-upload) [![Gitter chat](https://badges.gitter.im/apiaryio/s3-streaming-upload.png)](https://gitter.im/apiaryio/s3-streaming-upload)
 
-[s3-streaming-upload](https://github.com/apiaryio/s3-streaming-upload) is [node.js](http://nodejs.org) library that listens to your [stream](http://nodejs.org/docs/v0.8.9/api/stream.html) and upload its data to Amazon S3 using [MultiPartUpload API](http://docs.amazonwebservices.com/AmazonS3/latest/dev/sdksupportformpu.html).
+[s3-streaming-upload](https://github.com/apiaryio/s3-streaming-upload) is [node.js](http://nodejs.org) library that listens to your [stream](http://nodejs.org/docs/v0.8.9/api/stream.html) and upload its data to Amazon S3.
 
 It is heavily inspired by [knox-mpu](https://github.com/nathanoehlman/knox-mpu), but unlike it, it does not buffer data to disk and is build on top of [official AWS SDK](https://github.com/aws/aws-sdk-js) instead of knox.
 
+### Changes
```
Comment (Almad) [inline README.md]: @abtris I think you should also mention supported node.js versions

Comment (abtris) [inline README.md]: Good point.

**Pull request merged**